src/main/java/com/alkemy/ong/entity/ContactEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.alkemy.ong.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.SQLDelete;
+import org.hibernate.annotations.Where;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+@Entity
+@Table(name = "contact")
+@Getter
+@Setter
+@SQLDelete(sql = "UPDATE contact SET deletedAt = true WHERE id = ?")
```

Comment:
Shouldn't the SQL statement and clause be soft_delete?

Reply:
no the ticket asks me to name it like that and to use it as softdelete

Reply:
That's right I founded weird but has a different name. 
I think the table name should be in plural "contacts"